app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -554,4 +557,24 @@ public SynapseClient synapseClient() throws IOException {
         synapseClient.setApiKey(bridgeConfig().get("synapse.api.key"));
         return synapseClient;
     }
+
+    @Bean(name = "genericViewCache")
+    @Autowired
+    public ViewCache genericViewCache(CacheProvider cacheProvider) {
```

Comment:
Don't forget to remove the Component annotation from ViewCache. Otherwise, you'll end up with 3 ViewCaches, and the default one won't have its required properties.